infra/modules/main/backend.tf

```diff
@@ -42,7 +42,8 @@ module "backend_worker" {
   function_s3_bucket     = aws_s3_bucket.backend_code.id
   function_zipfile       = "backend-lambda.zip"
   function_handler       = "index.workerEntrypoint"
-  schedule_expression    = "rate(3 minutes)" # note: full cron expressions are also supported
+  function_timeout       = 60 * 5             # i.e. 5 minutes
```

Comment:
Just curious, how did you settle on these values?

Reply:
Pulled them from a hat for now. Let’s revisit when we’re doing more.